### ISSUE TITLE
Publish geolambda-python36 layer

### ISSUE THREAD

**Issue by matthewhanson** [developmentseed/geolambda#62]
The GeoLambda layer is usable in all runtimes, but there are also standard runtime specific libraries that are common for geo apps (e.g., GDAL python bindings, rasterio) which could be useful for some users as it would enable them to only have to provide a handler function if only using standard libraries.

Create a geolambda-python36 Lambda layer and publish it (include in CI deploys as well). The docker image is already built and published. 

For Lambda layers though the Python packages need to be in a python/ subdirectory in the deploy zip:
see AWS conventions: https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html#configuration-layers-path

**Comment by matthewhanson** [comment]
Looking for feedback (@vincentsarago @drewbo @sharkinsspatial) on what a common, yet small, set of common geospatial Python libraries should be included in a public Lambda layer. Some ideas:

- rasterio: clearly should be included
- GDAL Python bindings: if we have rasterio is it worth it to include this as well?
- pyproj: As geolambda already includes a build of the relevant version of PROJ.X I think this is worth including, I find I use it quite often.
- utm: if small enough this seems useful enough to include
- rio-cogeo: this seems like a generally useful library, my question is, if rasterio is already included what is the additional space requirements for rio-cogeo?

basically I'm looking for the geospatial libraries that have a small footprint that are dependent on GDAL/OGR, and are widely used enough vs their size that they are worth including.

**Comment by matthewhanson** [comment]
`utm` is just a small package with no geo dependencies, so it does not make sense to include.

`shapely` should be included.

**Comment by vincentsarago** [comment]
@matthewhanson 👍 for rasterio, gdal, pyproj and shapely. 

just make sure to install the right pyproj version for the right PROJ version
